tests/unittests/CoreGraphics.drawing/CGContextDrawing_ImageDrawingTests.cpp

```diff
@@ -0,0 +1,300 @@
+
+    CGAffineTransform flip = CGAffineTransformMakeScale(1, -1);
+    CGAffineTransform shift = CGAffineTransformTranslate(flip, 0, bounds.size.height * -1);
+    CGContextConcatCTM(context, shift);
+
+    CGContextDrawImage(context, bounds, image.get());
+}
+
+DISABLED_DRAW_TEST_F(CGContext, DrawAnImage, UIKitMimicTest<>) {
+    // Load an Image and draw it into the canvas context
+    auto drawingConfig = DrawingTestConfig::Get();
+
+    woc::unique_cf<CFStringRef> testFilename{ _CFStringCreateWithStdString(drawingConfig->GetResourcePath("jpg1.jpg")) };
+    woc::unique_cf<CGImageRef> image{ _CGImageCreateFromJPEGFile(testFilename.get()) };
+    ASSERT_NE(image, nullptr);
+
+    CGContextRef context = GetDrawingContext();
+    CGRect bounds = GetDrawingBounds();
+
+    CGAffineTransform flip = CGAffineTransformMakeScale(1, -1);
+    CGAffineTransform shift = CGAffineTransformTranslate(flip, 0, bounds.size.height * -1);
+    CGContextConcatCTM(context, shift);
+
+    CGContextDrawImage(context, bounds, image.get());
+}
+
+DRAW_TEST_F(CGContext, DrawAnImageWithOpacity, UIKitMimicTest<>) {
+    // Load an Image and draw it into the canvas context
+    auto drawingConfig = DrawingTestConfig::Get();
+
+    woc::unique_cf<CFStringRef> testFilename{ _CFStringCreateWithStdString(drawingConfig->GetResourcePath("png1.9.png")) };
+    woc::unique_cf<CGImageRef> image{ _CGImageCreateFromPNGFile(testFilename.get()) };
+    ASSERT_NE(image, nullptr);
+
+    CGContextRef context = GetDrawingContext();
+    CGRect bounds = GetDrawingBounds();
+
+    CGAffineTransform flip = CGAffineTransformMakeScale(1, -1);
+    CGAffineTransform shift = CGAffineTransformTranslate(flip, 0, bounds.size.height * -1);
+    CGContextConcatCTM(context, shift);
+
+    CGContextSetAlpha(context, 0.7);
+    CGContextDrawImage(context, bounds, image.get());
+}
+
+DRAW_TEST_F(CGContext, DrawAnImageWithInterpolationQuality, UIKitMimicTest<>) {
+    auto drawingConfig = DrawingTestConfig::Get();
+
+    woc::unique_cf<CFStringRef> testFilename{ _CFStringCreateWithStdString(drawingConfig->GetResourcePath("png1.9.png")) };
+    woc::unique_cf<CGImageRef> image{ _CGImageCreateFromPNGFile(testFilename.get()) };
+    ASSERT_NE(image, nullptr);
+
+    CGContextRef context = GetDrawingContext();
+    CGRect bounds = GetDrawingBounds();
+
+    CGContextSetInterpolationQuality(context, kCGInterpolationLow);
+    CGContextDrawImage(context, bounds, image.get());
+}
+
+DRAW_TEST_F(CGContext, DrawAnImageWithInterpolationQualityAndAlpha, UIKitMimicTest<>) {
+    auto drawingConfig = DrawingTestConfig::Get();
+    woc::unique_cf<CFStringRef> testFilename{ _CFStringCreateWithStdString(drawingConfig->GetResourcePath("png1.9.png")) };
+    woc::unique_cf<CGImageRef> image{ _CGImageCreateFromPNGFile(testFilename.get()) };
+    ASSERT_NE(image, nullptr);
+
+    CGContextRef context = GetDrawingContext();
+    CGRect bounds = GetDrawingBounds();
+
+    CGContextSetAlpha(context, 0.25);
```

Comment:
It would be good to make these parameterized over different alpha values/interpolation qualities